docker-for-aws/load-balancer.md

```diff
@@ -0,0 +1,83 @@
+---
+description: Load Balancer
+keywords: aws load balancer elb
+title: Docker for AWS Load Balancer Configuration
+---
+
+{% include d4a_buttons.md %}
+
+## How does it work?
+
+When you create a service any ports that are exposed with `-p` are automatically exposed through the platform load balancer:
+
+    $ docker service create --name nginx -p 80:80 nginx
+
+This will open up port 80 on the Elastic Load Balancer (ELB) and direct any traffic on that port, to your swarm service.
+
+## How can I configure my load balancer to support SSL/TLS traffic?
+
+Yes, you can. We are using [Amazons' ACM service](https://aws.amazon.com/certificate-manager/) which provides free SSL/TLS certificates, which can be used with ELBs. You will need to create a new certificate for your domain, and get the ARN for that certificate.
```

Comment:
But with the title above, this needs to be changed. Maybe just "Docker uses <link>, which provides free.... and can be used with ELBs.